BlockToolAdapter and configs error still persists + warnings

Related to [this issue.](https://github.com/codex-team/editor.js/issues/2771)

### ERRORS:

Error: node_modules/@editorjs/editorjs/types/api/tools.d.ts:1:34 - error TS2307: Cannot find module '@/types/tools/adapters/block-tool-adapter' or its corresponding type declarations.

`import { BlockToolAdapter } from '@/types/tools/adapters/block-tool-adapter';`

Error: node_modules/@editorjs/editorjs/types/tools/menu-config.d.ts:1:10 - error TS2305: Module '"../configs"' has no exported member 'PopoverItemDefaultBaseParams'.

`import { PopoverItemDefaultBaseParams, PopoverItemHtmlParams, PopoverItemParams, PopoverItemSeparatorParams, WithChildren } from "../configs";`

Error: node_modules/@editorjs/editorjs/types/tools/menu-config.d.ts:1:40 - error TS2305: Module '"../configs"' has no exported member 'PopoverItemHtmlParams'.

`import { PopoverItemDefaultBaseParams, PopoverItemHtmlParams, PopoverItemParams, PopoverItemSeparatorParams, WithChildren } from "../configs";`

Error: node_modules/@editorjs/editorjs/types/tools/menu-config.d.ts:1:63 - error TS2305: Module '"../configs"' has no exported member 'PopoverItemParams'.

`import { PopoverItemDefaultBaseParams, PopoverItemHtmlParams, PopoverItemParams, PopoverItemSeparatorParams, WithChildren } from "../configs";`

Error: node_modules/@editorjs/editorjs/types/tools/menu-config.d.ts:1:82 - error TS2305: Module '"../configs"' has no exported member 'PopoverItemSeparatorParams'.

`import { PopoverItemDefaultBaseParams, PopoverItemHtmlParams, PopoverItemParams, PopoverItemSeparatorParams, WithChildren } from "../configs";`

Error: node_modules/@editorjs/editorjs/types/tools/menu-config.d.ts:1:110 - error TS2305: Module '"../configs"' has no exported member 'WithChildren'.

`import { PopoverItemDefaultBaseParams, PopoverItemHtmlParams, PopoverItemParams, PopoverItemSeparatorParams, WithChildren } from "../configs";`

- Issue still persists even after update. All other packages are up to date.


### **FIX for menu-config.d.ts:**
menu-config.d.ts needs to have imports as:
`import { PopoverItemDefaultBaseParams, WithChildren, PopoverItemSeparatorParams, PopoverItemHtmlParams } from "@editorjs/editorjs/types/utils/popover";`
_This solves this issue._


tools.d.ts BlockToolAdapter can't find module from:

`import { BlockToolAdapter } from '@/types/tools/adapters/block-tool-adapter';`

Not sure how to solve this though. Can't find the BlockToolAdapter at all.

_These are the leftover issues._


### WARNINGS:
**Build also throws warnings related to editorJS:**

```
Warning:
C:/Company/Angular/ClientWeb/ClientApp/node_modules/@editorjs/editorjs/types/configs/conversion-config.ts is part of the TypeScript compilation but it's unused.
Add only entry points to the 'files' or 'include' properties in your tsconfig.

Warning: 
C:/Company/Angular/ClientWeb/ClientApp/node_modules/@editorjs/editorjs/types/data-formats/block-id.ts is part of the TypeScript compilation but it's unused.
Add only entry points to the 'files' or 'include' properties in your tsconfig.

Warning: 
C:/Company/Angular/ClientWeb/ClientApp/node_modules/@editorjs/editorjs/types/events/block/Base.ts is part of the TypeScript compilation but it's unused.
Add only entry points to the 'files' or 'include' properties in your tsconfig.

Warning: 
C:/Company/Angular/ClientWeb/ClientApp/node_modules/@editorjs/editorjs/types/events/block/BlockAdded.ts is part of the TypeScript compilation but it's unused.
Add only entry points to the 'files' or 'include' properties in your tsconfig.

Warning: 
C:/Company/Angular/ClientWeb/ClientApp/node_modules/@editorjs/editorjs/types/events/block/BlockChanged.ts is part of the TypeScript compilation but it's unused.
Add only entry points to the 'files' or 'include' properties in your tsconfig.

Warning: 
C:/Company/Angular/ClientWeb/ClientApp/node_modules/@editorjs/editorjs/types/events/block/BlockMoved.ts is part of the TypeScript compilation but it's unused.
Add only entry points to the 'files' or 'include' properties in your tsconfig.

Warning: 
C:/Company/Angular/ClientWeb/ClientApp/node_modules/@editorjs/editorjs/types/events/block/BlockRemoved.ts is part of the TypeScript compilation but it's unused.
Add only entry points to the 'files' or 'include' properties in your tsconfig.

Warning: 
C:/Company/Angular/ClientWeb/ClientApp/node_modules/@editorjs/editorjs/types/events/block/index.ts is part of the TypeScript compilation but it's unused.
Add only entry points to the 'files' or 'include' properties in your tsconfig.

Warning: 
C:/Company/Angular/ClientWeb/ClientApp/node_modules/@editorjs/editorjs/types/utils/popover/popover-event.ts is part of the TypeScript compilation but it's unused.
Add only entry points to the 'files' or 'include' properties in your tsconfig.

Warning: 
C:/Company/Angular/ClientWeb/ClientApp/node_modules/@editorjs/editorjs/types/utils/popover/popover-item-type.ts is part of the TypeScript compilation but it's unused.
Add only entry points to the 'files' or 'include' properties in your tsconfig.
```

--- a/docs/CHANGELOG.md
+++ b/docs/CHANGELOG.md
@@ -1,5 +1,9 @@
 # Changelog
 
+### 2.30.5
+
+– `Fix` – Fix exported types
+
 ### 2.30.4
 
 - `Fix` - Tool's exporting types added

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@editorjs/editorjs",
-  "version": "2.30.4",
+  "version": "2.30.5",
   "description": "Editor.js — open source block-style WYSIWYG editor with JSON output",
   "main": "dist/editorjs.umd.js",
   "module": "dist/editorjs.mjs",

--- a/types/api/tools.d.ts
+++ b/types/api/tools.d.ts
@@ -1,4 +1,4 @@
-import { BlockToolAdapter } from '@/types/tools/adapters/block-tool-adapter';
+import { BlockToolAdapter } from '../tools/adapters/block-tool-adapter';
 
 /**
  * Describes methods for accessing installed Editor tools

--- a/types/tools/menu-config.d.ts
+++ b/types/tools/menu-config.d.ts
@@ -1,4 +1,4 @@
-import { PopoverItemDefaultBaseParams, PopoverItemHtmlParams, PopoverItemParams, PopoverItemSeparatorParams, WithChildren } from "../configs";
+import { PopoverItemDefaultBaseParams, PopoverItemHtmlParams, PopoverItemSeparatorParams, WithChildren } from '../utils/popover';
 
 /**
  * Menu configuration format.

